backend/src/main/kotlin/org/loculus/backend/config/Config.kt

```diff
@@ -47,7 +47,16 @@ const val SINGLE_REFERENCE_GENOME_KEY = "singleReference"
 
 typealias Suborganism = String
 
-data class InstanceConfig(val schema: Schema, val referenceGenomes: Map<Suborganism, ReferenceGenome>)
+data class InstanceConfig(
+    val schema: Schema,
+    val referenceGenomes: Map<Suborganism, ReferenceGenome>,
+    /**
+     * In the single pathogen case, this will be equal to the single entry in `referenceGenomes`.
+     * In the multi-pathogen case, this will a flattened version of the `referenceGenomes` map,
+     * where the sequences of the suborganisms are merged into a list with names prefixed with the suborganism name.
+     */
+    val mergedReferenceGenome: ReferenceGenome,
```

Comment:
So we wanted to just start by having the ebola viruses as a multi-pathogen and keeping the other viruses as is - how does this work now? What will the cchf sequences be in the `mergedReferenceGenome`? As it looks like we will be using this now for the processedData validation